girder/girder_large_image/web_client/vue/components/CompositeLayers.vue

```diff
@@ -57,6 +58,44 @@ export default {
                     usedColors.push(chosenColor)
                 }
             })
+            this.fetchHistograms()
+        },
+        fetchHistograms() {
+            if (this.layerMap) {
+                // layers are channels; each layer has a frame delta
+                this.layers.forEach((layer) => {
+                    restRequest({
+                        type: 'GET',
+                        url: 'item/' + this.itemId + '/tiles/histogram',
+                        data: {
+                            frame: this.currentFrame + this.compositeLayerInfo[layer].framedelta,
+                            width: 1024,
+                            height: 1024,
+                            bins: 512,
```

Comment:
We also need `resample: false` in the query, or the scaling step may end up with histograms on 8-bit data even when the source is not 8-bit.  We might want to have a single place for these parameters rather than repeat them in the bands section.

Reply:
Changed in [b44d581](https://github.com/girder/large_image/pull/1175/commits/b44d581d3a457415036a8f2d46bbcb8d8e2dfbe0)